lighty-modules/lighty-gnmi/lighty-gnmi-test/src/test/java/io/lighty/modules/gnmi/test/gnmi/rcgnmi/GnmiConnectionITTest.java

```diff
@@ -513,4 +539,16 @@ public void connectDeviceWithIncorrectCredentialsTest()
             });
     }
 
+    private void assertExistingAndEmptyGnmiTopology() throws IOException, InterruptedException {
+        final HttpResponse<String> getGnmiTopologyResponse = sendGetRequestJSON(GNMI_TOPOLOGY_PATH);
+        assertEquals(HttpURLConnection.HTTP_OK, getGnmiTopologyResponse.statusCode());
+        final JSONArray topologies =
+            new JSONObject(getGnmiTopologyResponse.body()).getJSONArray("network-topology:topology");
+        assertEquals(1, topologies.length());
+        final JSONObject gnmiTopologyJSON = topologies.getJSONObject(0);
+        LOG.info("Empty gnmi-topology check response: {}", gnmiTopologyJSON);
+        assertEquals("gnmi-topology", gnmiTopologyJSON.getString("topology-id"));
+        assertThrows(JSONException.class, () -> gnmiTopologyJSON.getJSONArray("node"));
+    }
+
```

Comment:
You should not change something that is not described in commit message - also tests are written to be verbose and were purposely not packaged into methods (but if you want to do that, methods should not contain asserts, they should be visible in tests and it should be clear what is asserting as output from the method), so please revert this changes with this method